app/index.js

```diff
@@ -118,17 +118,26 @@ var Generator = module.exports = yeoman.generators.Base.extend({
 		var configsPath = path.join(this.sourceRoot(), configs.server);
 		var destPath = this.destinationRoot();
 
+		// node server config is fetched via npm
+		if (configs.server == 'node') {
+			this.copy('package.json');
+			this.copy('index.js');
+			done();
+
+			this.npmInstall(['h5bp'], {
```

Comment:
I wonder if we shouldn't use `installDependencies()` here. Also let's support a `--skip-install` option, nearly every yeoman generator has that.